src/h5web/visualizations/line/hooks.ts

```diff
@@ -25,8 +23,13 @@ export function useDataDomain(): Domain | undefined {
 
 export function useDataPoints(): Vector3[] | undefined {
   const { data } = useProps();
-  const { scale: abscissaScale } = useAbscissaScale();
-  const { scale: ordinateScale } = useOrdinateScale();
+
+  const { abscissa, ordinate } = useContext(AxisSystemContext);
+  const { size } = useThree();
+  const { width, height } = size;
+
+  const abscissaScale = getAxisScale(abscissa, width);
+  const ordinateScale = getAxisScale(ordinate, height);
```

Comment:
I decided to remove the `useAbscissaScale` and `useOrdinateScale` now that the interface of `getAxisScale` is a lot simpler. I no longer think that the convenience is worth the added complexity.